RELEASING.md

```diff
@@ -49,6 +55,7 @@ The resulting localized resources are added/updated in the repository periodical
 ### Post Release
 
 - [ ] Publish blog post
+- [ ] Update news files
```

Comment:
`, if needed.`? zapbot already updates them.

Reply:
True, although I like the "Welcome to X" news item for the latest release - will tweek..

Reply:
Tweaked..